fouleggs/src/de/up/hpi/informationsystems/fouleggs/dactors/AdminSession.scala

```diff
@@ -0,0 +1,176 @@
+package de.up.hpi.informationsystems.fouleggs.dactors
+
+import akka.actor.Actor.Receive
+import akka.actor.{Actor, ActorLogging, ActorRef, ActorSystem, Props}
+import de.up.hpi.informationsystems.adbms.Dactor
+import de.up.hpi.informationsystems.adbms.protocols.DefaultMessagingProtocol
+import de.up.hpi.informationsystems.adbms.record.Record
+import de.up.hpi.informationsystems.adbms.record.ColumnCellMapping._
+import de.up.hpi.informationsystems.adbms.relation.Relation
+
+object AdminSession {
+
+  final case class Up()
+
+  object AddCastToFilm {
+    final case class Request(personId: Int, filmId: Int, roleName: String)
+    final case class Success(personId: Int, filmId: Int, roleName: String)
+    final case class Failure(e: Throwable)
```

Comment:
Those messages do not extend our `RequestResponseProtocol`. This is nothing bad in general, but they look like they should be subtypes of it (`AddCastToFilm` defines our Message-Trinity: `Request`, `Success`, `Failure`)!